### ISSUE TITLE
TestISOConfigPrepare_ISOChecksumURL fails on Windows when running make test

### ISSUE THREAD

**Issue by metalmolly** [hashicorp/packer#3215]
`make test` fails on `TestISOConfigPrepare_ISOChecksumURL` as Windows handles file: URI somewhat different then nix systems. Tested with GO 1.5.3 on Windows 8.1 with 32bbe55f2b6f71837cec3568e2582a459711a2e4

```
c:\Users\MarkS\go\src\github.com\mitchellh\packer>make test TEST=./common/ 
go get github.com/mitchellh/gox
go get golang.org/x/tools/cmd/stringer
go get golang.org/x/tools/cmd/vet
...
2016/02/16 22:47:57 [DEBUG] Using local file: /test-fixtures/fileurl/cake
2016/02/16 22:47:57 Verifying checksum of test-fixtures/fileurl/cake
--- FAIL: TestISOConfigPrepare_ISOChecksumURL (0.00s)
        iso_config_test.go:79: should not have error: [open : The system cannot find the file specified.]
...
FAIL
FAIL    github.com/mitchellh/packer/common      0.146s
Makefile:46: recipe for target 'test' failed
make: *** [test] Error 1
```

**Comment by cbednarski** [comment]
Thanks for the report!

**Comment by arizvisa** [comment]
This looks like it'd be close to a dupe of whatever would help fix #2377.

**Comment by SwampDragons** [comment]
Closing since I think this was fixed on master some time ago.
